### ISSUE TITLE
Some adjustments for Text-to-speech

### ISSUE THREAD

**Issue by hssm** [ankidroid/Anki-Android#1361]
Originally reported on Google Code with ID 458

```
Well, I know that TTS is a new feature, already "a child", but here's some suggestions
for enhance it's usability in the future:

1. Choosing which fields should be read with TTS. Actually, there's only the option
of choosing the language for TTS in each field, but there isn't any option of "discarding"
TTS for one field.
2. Well, point 1 isn't totally true, since the "back" button can be pressed to discard
TTS only for the present field. The problem is that this action isn't recorded for
the rest of the session, and if we want one field without TTS, we must press "back"
each time TTS attempts to read this field.
3. The option to activate TTS per deck. Some users have decks which they want to use
TTS, and some they don't want.

So, in short: 
1. A button of "discard" in the emerging menu of TTS language selection.
2. Recording of "discard" selection for the rest of the session.
3. Activate/deactivate TTS per deck.

```

Reported by `Villatonil` on 2011-02-23 09:39:20
